### DISCUSSION TITLE
Distil-Whisper models

### DISCUSSION THREAD

**Discussion by Purfview** [Purfview/whisper-standalone-win#180] [Announcements]
[Distil-Whisper](https://github.com/huggingface/distil-whisper) models supported since r172.1.

`distil-large-v3`
`distil-large-v2`
`distil-medium.en`
`distil-small.en`

Distil models for Faster-Whisper require ~twice less memory and they are roughly ~twice faster, when transcription accuracy is somewhere between an original model and a next smaller one.

Note1: These are English only models.
Note2: Just use `turbo` model, it's better than distill and it's multilingual.

**Comment by ghost** [comment]
ayyooo whats ur thought rn should we switch to them

**Reply by Purfview** [reply]
If you don't have hardware issues running the original models then there is no reason to use distill models, as they are less accurate.

**Comment by pto2k** [comment]
Hi, besides the official/distilled Whisper models, can other fine-tune models be used?
For example: https://huggingface.co/BELLE-2/Belle-whisper-large-v3-zh
Thank you!

**Reply by Purfview** [reply]
Yes, finetuned models can be used, but your linked model is not faster-whisper model.

**Comment by BanQuanChung** [comment]
Could you please make a distil-large-v2 model for the Vietnamese langugage?
Thanks.

**Reply by Purfview** [reply]
Just use `turbo` model, it's better than distill and it's multilingual.
